src/Jupyter/Visualization/MeasurementHistogramContent.cs

```diff
@@ -19,6 +20,12 @@ public class MeasurementHistogramContent : MessageContent
         /// Information about the state to be displayed.
         /// </summary>
         [JsonProperty("state")]
-        public DisplayableState State { get; set; } = new DisplayableState();
+        public CommonNativeSimulator.DisplayableState State { get; set; } = new CommonNativeSimulator.DisplayableState();
+
+        /// <summary>
+        /// Unique identifier of the instance.
+        /// </summary>
+        [JsonProperty("id")]
+        public string Id { get; set; } = $"{System.Guid.NewGuid()}";
```

Comment:
Ah, right, I forgot we already had this! This should be what we need to replace `DivId` in `DisplayableState`.